lib/Class/Method/Modifiers.pm

```diff
@@ -37,9 +44,15 @@ sub install_modifier {
     return _fresh($into, $code, @names) if $type eq 'fresh';
 
     for my $name (@names) {
-        my $hit = $into->can($name) or do {
+
+        my $orig = $name;
+
+        $name = _overloaded_op_method( $into, $name )
+          if $OP{$name};
```

Comment:
This may be reasonable for things like `""` that will never be a normal method name, it isn't workable for those that are valid identifiers (`cmp`, `sin`, `bool`, etc).